static/app/views/settings/components/teamSelect.tsx

```diff
@@ -92,8 +92,13 @@ function TeamSelect({
   };
 
   // Only show options that aren't selected in the dropdown
+  // and that aren't idp-provisioned
```

Comment:
idp provisioned teams can only have users added to them through the identity provider

Reply:
@cathyteng17 can you check that this component is only used for assigning `Members` into `Teams`. 

It might make sense to add a comment or change the name of the component to make it explicitly clear that this is cannot be used for generic assignment of "X" to `Teams`, because of the IdP-provisioned restriction.

Reply:
this component is also used to assign a project to a team... https://github.com/getsentry/sentry/blob/0dbe496f9d856824206072b2a192b61c1ca8f5da/static/app/views/settings/project/projectTeams.tsx#L149

Reply:
i added an extra prop to make sure adding an idp-provisioned team is only disabled if this component is being used for org members adding teams